### PR TITLE
Added content to the splash page

### PR DESCRIPTION
**Pull request by Yaomimi**
Might need to edit it again

# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

* [ ] `package.json` is up-to-date and `@ecl/[system]-base` is part of the dependencies
* [ ] I have checked the dependencies
* [ ] I have given the fractal status “ready” to my component
* [ ] I have declared `@define mycomponent` in the SCSS file
* [ ] I have specified `margin: 0;` on the CSS component
* [ ] I have provided tests
* [ ] I follow the naming guidelines
* [ ] the component supports composition
* [ ] there are no hardcoded strings (all content come from the context)
* [ ] I have filled the README.md file (at least a few lines)
* [ ] My component is listed in the root README
* [ ] My PR has the right label(s)

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit b27be2ec2a33ceda32297b676ebc8ac3d86218ec

https://deploy-preview-752--europa-component-library.netlify.com

File: website/src/pages/index.md [hunk 1]
```diff
@@ -1,11 +1,5 @@
-# Component Library
+# Europa Component Library
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut convallis
-metus non lectus dignissim, ac viverra elit convallis. Pellentesque
-tempus purus id facilisis lacinia.
+The **Europa Component Library** (ECL) is a living style guide containing the design standards and visual assets for the European Commission websites hosted under the ec.europa.eu domain and the European Union website hosted under the europa.eu domain.
```
Comment (yhuard) [inline website/src/pages/index.md]: `Commission websites` or  `Commission's websites`?

File: website/src/pages/index.md [hunk 2]
```diff
@@ -1,11 +1,5 @@
...
 
-Nulla consequat suscipit commodo. Fusce sollicitudin orci id volutpat
-tempor. Duis vestibulum, diam vel gravida dapibus, odio velit eleifend
-felis, id elementum dolor nisi id arcu.
-
-Select between the available libraries:
+Get started using the ECL components for your projects. Choose one the component libraries below.
```
Comment (yhuard) [inline website/src/pages/index.md]: `one the`? I think there's a missing "of" here

File: website/src/pages/index.md (hunk 1 above)
Comment (Yaomimi) [reply website/src/pages/index.md]: Our content writer wrote it Commission websites. So it is correct.

File: website/src/pages/index.md (hunk 2 above)
Comment (Yaomimi) [reply website/src/pages/index.md]: Correct. Thanks for pointing it out.

**Pull request closed without merging**